### PR TITLE
add global_conf

### PR DESCRIPTION
**Pull request by dkulykovmirantis**

### CODE REVIEW

File: mk_verificator/utils/__init__.py [hunk 1]
```diff
@@ -38,7 +38,18 @@ def get_groups(config):
...
 def get_configuration(path_to_test):
+    """global conf is minor, but local conf is major."""
+
     test_folder = os.path.dirname(os.path.abspath(path_to_test))
     config_file = '/'.join([test_folder, "config.yaml"])
     config = yaml.load(open(config_file, 'r'))
-    return config
+
+    global_config_folder = '/'.join(os.path.dirname(
+        os.path.abspath(__file__)).split('/')[:-1])
```
Comment (msenin94) [inline mk_verificator/utils/__init__.py]: frankly speaking, better do not use split here. I would prefer kinda of:

just an example:
global_config_folder = os.path.join(current_file_path, '../../..')

**Comment by vkhalin** [comment]
plz fix travis stuff. For me looks good

File: mk_verificator/utils/__init__.py [hunk 2]
```diff
@@ -38,7 +38,18 @@ def get_groups(config):
 
 
 def get_configuration(path_to_test):
+    """global conf is minor, but local conf is major."""
```
Comment (msenin94) [inline mk_verificator/utils/__init__.py]: okies, let's rephrase it:

function returns configuration for environment and for test if it's specified.

File: mk_verificator/utils/__init__.py [hunk 3]
```diff
@@ -38,7 +38,18 @@ def get_groups(config):
...
+
```
Comment (msenin94) [inline mk_verificator/utils/__init__.py]: then let's reorganize a bit our logic:
1) load global_config
2) check if local config for test
3)
if local_config:
   global_config.update(local_config)

In my vision it makes sense

File: mk_verificator/utils/__init__.py [hunk 4]
```diff
@@ -38,7 +38,19 @@ def get_groups(config):
 
 
 def get_configuration(path_to_test):
-    test_folder = os.path.dirname(os.path.abspath(path_to_test))
-    config_file = '/'.join([test_folder, "config.yaml"])
-    config = yaml.load(open(config_file, 'r'))
-    return config
+    """function returns configuration for environment
+
```
Comment (msenin94) [inline mk_verificator/utils/__init__.py]: newline is not needed here

**Pull request merged**